src/dotnet/commands/dotnet-new/CSharp_Web/project.json.template

```diff
@@ -0,0 +1,108 @@
+{
+  "userSecretsId": "aspnet-WebApplication-0799fe3e-6eaf-4c5f-b40e-7c6bfd5dfa9a",
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
```

Comment:
What's the plan for keeping these dependencies up to date?  Is our goal to ship RC2 templates?  Or do we want to ship RTM templates?

If we want RTM templates, how are these templates going to be updated to the final ASP.NET build?

Also, there is a script in our repo that automatically updates these dependencies when new CoreFX builds come out.  You are going to want to skip this template so it doesn't get updated automatically.  See https://github.com/dotnet/cli/blob/rel/1.0.0/build_projects/update-dependencies/UpdateFilesTargets.cs#L93